samples/gcp-storage/src/storageHandler.js

```diff
@@ -0,0 +1,22 @@
+const { App, HTTPBindingMiddleware, StorageMiddleware } = require("core");
+const { GcpModule } = require("gcp");
+const { streamToString } = require("./streamToString");
+const { bucketName } = require("../config.google.json");
+
+const app = new App(new GcpModule());
+
+module.exports.handler = app.use(
+  [HTTPBindingMiddleware(), StorageMiddleware()],
+  async (context) => {
+    const { name } = context.req.query;
```

Comment:
The query type is StringParams:

![image](https://user-images.githubusercontent.com/47789756/100930674-2db6f280-34c8-11eb-9391-6b90e496077b.png)

You could change it to `const { name } = context.req.query.toJson()`?

![image](https://user-images.githubusercontent.com/47789756/100930880-75d61500-34c8-11eb-8604-812f14d08abc.png)

Reply:
Done in a8a0c1e7acf6e7e3f64dfe26c570ef0224198c8d